tests/e2e/deployment-test.sh

```diff
@@ -159,7 +159,7 @@ kubectl -n dynatrace get pods
 
 # Generate load on GC Function
 for i in {1..5}; do
-  curl "https://us-central1-${GCP_PROJECT_ID}.cloudfunctions.net/sample_app?deployment_type=${DEPLOYMENT_TYPE}&build_id=${TRAVIS_BUILD_ID}" \
+  curl "https://us-central1-${GCP_PROJECT_ID}.cloudfunctions.net/e2e_test_sample_app?deployment_type=${DEPLOYMENT_TYPE}&build_id=${TRAVIS_BUILD_ID}" \
```

Comment:
@mswiatkowska if you are using variable `"${CLOUD_FUNCTION_NAME}"` use it also here instead of hardcoded name